phase-4-explainer.md

```diff
@@ -153,42 +150,80 @@ interface ComputedAccessibleNode {
     readonly attribute unsigned long? rowSpan;
     readonly attribute long? setSize;
 
+    readonly attribute float? valueNow;
+    readonly attribute float? valueMin;
+    readonly attribute float? valueMax;
+
+    readonly attribute DOMString? autocomplete;
+    readonly attribute DOMString? checked;
+    readonly attribute DOMString? keyShortcuts;
+    readonly attribute DOMString? name;
+    readonly attribute DOMString? placeholder;
+    readonly attribute DOMString? role;
+    readonly attribute DOMString? roleDescription;
+    readonly attribute DOMString? valueText;
+
     readonly attribute ComputedAccessibleNode? parent;
     readonly attribute ComputedAccessibleNode? firstChild;
     readonly attribute ComputedAccessibleNode? lastChild;
     readonly attribute ComputedAccessibleNode? previousSibling;
     readonly attribute ComputedAccessibleNode? nextSibling;
 
-    readonly attribute boolean? atomic;
-    readonly attribute boolean? busy;
-    readonly attribute boolean? disabled;
-    readonly attribute boolean? modal;
-    readonly attribute boolean? readOnly;
-
-    readonly attribute float? valueNow;
-    readonly attribute float? valueMin;
-    readonly attribute float? valueMax;
-
     [CallWith=ScriptState] Promise ensureUpToDate();
 };
 ```
 
-TODO(meredithl): insert link to full list of potential properties in spec.
-
+See the [proposed Phase 4 Spec](spec/phase4.html) for a full list of properties available to be implemented.
 
 ### 2. Rejection Case
-If we are returning a promise when requesting a `ComputedAccessibleNode`, what is the rejection case?
+If we are returning a promise when requesting a `ComputedAccessibleNode`, what is the rejection case? What does it mean for a `ComputedAccessibleNode` to be rejected? This may be discovered as other open questions are answered.
 
 ### 3. Traversing across Accessible Trees
 Currently Phase 4 is proposed as containing `ComputedAccessibleNode`s within a single Frame, but in future the capability to explore from one Accessible Tree in one Frame to another.
-- TODO: Use cases for this?
-- Considerations? How to get from one "root" to "another"?
+- Considerations?
```

Comment:
This section might need some more expansion, I don't really understand this section tbh.

Reply:
We can chat about it later, along with all that promise resolver fun.